group-size-refinement-list/react-instantsearch/src/lib/GroupSizeRefinementList.js

```diff
@@ -43,12 +60,40 @@ class GroupSizeRefinementList extends Component {
                                     ) : (
                                             item.label
                                         )}
-                                    <li className="ais-GroupSizeRefinementCount">{item.count}</li>
+                                    <span className="ais-GroupSizeRefinementCount">{item.count}</span>
                                 </li>
                             </a>
                         ))}
                     </ul>
                 ))}
+                {selectedSizes !== [] &&
```

Comment:
@Haroenv Here I'm pretty much duplicating the dom to display selected sizes. Is there any other way to do it ?
https://cl.ly/c708ad3b7b0d <- It's juste here to still display selected sizes that should be hidden when clicking show less

Reply:
@valentindotxyz ?